ballerina/tests/click-test.bal

```diff
@@ -0,0 +1,381 @@
+}
+
+@test:Config {
+    groups: ["account"]
+}
+function testServiceInfo() returns error? {
+    ServiceInformation expectedPayload = {
+        buildVersion: "23.4.0.266 (apiclick2023.10.29.266+b6661c114fe2)",
+        linkedSites: ["https://demo.docusign.net"],
+        serviceVersions: [
+            {
+                version:"v1",
+                versionUrl:"https://demo.docusign.net/clickapi/v1"
+            }
+        ]
+    };
+    ServiceInformation response = check docuSignClient->/service_information;
+    test:assertEquals(response, expectedPayload);
+}
+
+@test:Config {
+    groups: ["account"]
+}
+function testCreateClickWrap() returns error? {
+    string base64Encoded = array:toBase64(check io:fileReadBytes("resources/README.pdf"));
+    string clickwrapName = "ReturnPolicy";
+    ClickwrapRequest payload =  {
+        clickwrapName: clickwrapName,
+        documents: [
+            {
+                documentName: "Test Doc",
+                documentBase64: base64Encoded,
+                fileExtension: "pdf"
+            }
+        ],
+        displaySettings: {
+            displayName: "Return Policy",
+            consentButtonText: "I Agree",
+            downloadable: true,
+            format: "modal",
+            requireAccept: true,
+            documentDisplay: "document",
+            sendToEmail: true 
+        }
+    };
+    ClickwrapVersionSummaryResponse response = check docuSignClient->/v1/accounts/[accountId]/clickwraps.post(payload);
+    test:assertEquals(response.clickwrapName, clickwrapName);
+    _ = check docuSignClient->/v1/accounts/[accountId]/clickwraps/[<string>response.clickwrapId].delete();
+}
+
+// @test:Config {
+//     groups: ["account"]
+// }
+// function testUpdateClickWrap() returns error? {
+//     string base64Encoded = array:toBase64(check io:fileReadBytes("resources/README.pdf"));
+//     string clickwrapName = "ReturnPolicy";
+//     ClickwrapRequest payload =  {
+//         clickwrapName: clickwrapName,
+//         documents: [
+//             {
+//                 documentName: "Test Doc",
+//                 documentBase64: base64Encoded,
+//                 fileExtension: "pdf"
+//             }
+//         ],
+//         displaySettings: {
+//             displayName: "Return Policy",
+//             consentButtonText: "I Agree",
+//             downloadable: true,
+//             format: "modal",
+//             requireAccept: true,
+//             documentDisplay: "document",
+//             sendToEmail: true 
+//         }
+//     };
+//     ClickwrapVersionSummaryResponse response = check docuSignClient->/v1/accounts/[accountId]/clickwraps.post(payload);
+//     ClickwrapVersionSummaryResponse updatedResponse = check docuSignClient->/v1/accounts/[accountId]/clickwraps/[<string>response.clickwrapId].put({
+//         transferFromUserId: response.ownerUserId,
+//         transferToUserId: ""
+//     });
+//     test:assertEquals(updatedResponse.accountId, accountId);
+// }

```

Comment:
Let's remove the commented out code.

Reply:
Fixed in [2adfefa](https://github.com/ballerina-platform/module-ballerinax-docusign.dsclick/pull/1/commits/2adfefa9bd1f9d8fd566dbf634d363d21553a08a)